.changeset/early-olives-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+"effect": minor
```

Comment:
```suggestion
"effect": patch
```

While Micro is experimental I think we can do patch releases for new features.

Reply:
Yup